File: java/easiest.java
import java.util.Scanner;

public class easiest {
	static int sumOfDigits(int num) {
		int sum = 0;
		while (num > 0) {
			sum += num % 10;
			num /= 10;
		}
		return sum;
	}

	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		int N;
		while ((N = scan.nextInt()) != 0) {
			int result = -1;
			int sum = sumOfDigits(N);
			for (int i = 11; i < Integer.MAX_VALUE; i++)
				if (sum == sumOfDigits(N * i)) {
					result = i;
					break;
				}
			System.out.println(result);
		}
		scan.close();
	}
}

File: ts/src/easiest.ts
import readline = require('readline')

function sumOfDigits (num: number): number {
  let sum = 0
  while (num > 0) {
    sum += num % 10
    num = Math.floor(num / 10)
  }
  return sum
}

function minimumMultiplier (num: number): number {
  const sum = sumOfDigits(num)
  for (let i = 11; i < Number.MAX_SAFE_INTEGER; i++) {
    if (sum === sumOfDigits(num * i)) {
      return i
    }
  }
  return -1
}

readline
  .createInterface({
    input: process.stdin,
    output: process.stdout
  })
  .on('line', (line: string) => {
    if (line === '0') {
      process.exit(0)
    }
    console.log(minimumMultiplier(parseInt(line)))
  })
  .on('close', () => {
    process.exit(0)
  })
